benches/perf_ci/pyperformance/bm_richards.py

```
@@ -0,0 +1,423 @@
+        if t.isTaskHoldingOrWaiting():
+            t = t.link
+        else:
+            if tracing:
+                trace(chr(ord("0") + t.ident))
+            t = t.runTask()
+
+
+class Richards(object):
+
+    def run(self, iterations):
+        for i in range(iterations):
```

Ruff (0.16.1)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)